pkgs/by-name/mg/mgitstatus/package.nix

```diff
@@ -0,0 +1,42 @@
+{ fetchFromGitHub
+, lib
+, mgitstatus
+, stdenv
+, testers
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "mgitstatus";
+  version = "2.2";
+
+  src = fetchFromGitHub {
+    owner = "fboender";
+    repo = "multi-git-status";
+    rev = finalAttrs.version;
+    hash = "sha256-jzoX7Efq9+1UdXQdhLRqBlhU3cBrk5AZblg9AYetItg=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm 755 mgitstatus $out/bin/${finalAttrs.pname}
+    install -Dm 644 mgitstatus.1 $out/share/man/man1/${finalAttrs.pname}.1
+
+    runHook postInstall
+  '';
```

Comment:
```suggestion
  installFlags = [
    "PREFIX=$(out)"
  ];
```

Reply:
So this will copy everything from `$src` and put it in `$out`?

Reply:
Yes, `make install PREFIX=\$(out)` is run in the installPhase https://github.com/fboender/multi-git-status/blob/2709401fa64c890c21388dcef5de1c7c05521450/Makefile#L22